bridge/gitlab/import.go

```diff
@@ -269,6 +323,9 @@ func (gi *gitlabImporter) ensureNote(repo *cache.RepoCache, b *cache.BugCache, n
 		NOTE_UNLOCKED,
 		NOTE_MENTIONED_IN_ISSUE,
 		NOTE_MENTIONED_IN_MERGE_REQUEST:
+
+		reason := fmt.Sprintf("unsupported note type: %v", noteType)
```

Comment:
Maybe better readable reasons instead of just passing the `noteType` which is an integer here

Reply:
done